swarmer/src/main/kotlin/com/gojuno/swarmer/Args.kt

```diff
@@ -46,7 +46,7 @@ sealed class Commands {
                     description = "Package of the system image for this AVD (e.g.'system-images;android-25;google_apis;x86') to pass to `avdmanager create avd --package`.",
                     order = 2
             )
-            var pakage: String = "",
+            var `package`: String = "",
```

Comment:
@artem-zinnatullin I forgot to patent it... So I can nothing to do with it now. Hail the King Artur!